colcon-build/action.yaml

```diff
@@ -75,6 +75,9 @@ runs:
         else
             rosdep update
         fi
+        if [ -f "/opt/autoware/setup.sh" ]; then
```

Comment:
I think we should pass this path as an optional argument rather than having an hard coded path.

Reply:
It's a good suggestion. Fixed https://github.com/autowarefoundation/autoware-github-actions/pull/337/commits/5e6103544d227736a5bef00a0c8c7422e350f15e.